benchmarks/tpch/sf100.json

```diff
@@ -1,4 +1,4 @@
 {
   "description": "Specifies the schema for TPCH scale factor 100 GB.",
-  "schema": "tpch_sf100_parquet"
+  "schema": "tpch_sf100_parquet_v2"
```

Comment:
This is the wrong schema. tpch_sf100_parquet_v2 does not exist. We want to change the corresponding tpcds schema